runtime/executor/src/test/java/org/apache/nemo/runtime/executor/data/BlockTest.java

```diff
@@ -42,6 +42,7 @@
 public final class BlockTest {
   private Serializer serializer;
   private Map<Integer, List<Integer>> testData;
+  private static final MemoryPoolAssigner memoryPoolAssigner = new MemoryPoolAssigner(1, 32);
```

Comment:
This should be initialized upon setUp() (`@Before`), similar to the serializer variable.
If not, we're reusing the variable across `@Test`s, which might cause undesirable side effects.